packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -2277,9 +2289,9 @@ describe('NetworkController', () => {
     });
   });
 
-  describe('setRpcTarget', () => {
-    describe('given only an RPC target and chain ID', () => {
-      it('updates the provider config in state with the RPC target and chain ID, clearing any existing ticker and nickname', async () => {
+  describe('setActiveNetwork', () => {
+    describe('given only a networkConfigurationId', () => {
```

Comment:
Nit: Is this describe block still necessary? It looks like the only one under `setRpcTarget`.

Reply:
nope! removing!

Reply:
done here: https://github.com/MetaMask/core/pull/1064/commits/0124a625a57d81ae00c1dcaa03e880c62f6ff14e#diff-3439880e0b961116f2219977efca0fe970b9fa919ac62b5ad934c2274edf206dL2293